scripts/o3de/o3de/manifest.py

```diff
@@ -236,6 +240,12 @@ def is_gem_subdirectory(subdir_files):
 
 
 # Data query methods
+def get_this_engine() -> dict:
+    json_data = load_o3de_manifest()
+    engine_data = find_engine_data(json_data)
+    return engine_data
+
```

Comment:
This is not a good name for this function.
"get_this_engine" what? engine.json data?, path?, engine_name?

This at the very least should be called `get_this_engine_json_data

Reply:
Also this function isn't actually used and is redundant with the [print_registration.print_this_engine](https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/print_registration.py#L42-L48) method

Reply:
I'll remove it